src/queries/SUE/requests.tsx

```diff
@@ -20,3 +20,55 @@ export const requests = async () => {
     );
   });
 };
+
+export const postRequests = async (data: any) => {
+  const formData = new FormData();
+  formData.append('address_string', data?.addressString);
+  formData.append('description', data?.description);
+  formData.append('email', data?.email);
+  formData.append('first_name', data?.firstName);
+  formData.append('jurisdiction_id', jurisdictionId);
+  formData.append('last_name', data?.lastName);
+  formData.append('lat', data?.lat);
+  formData.append('long', data?.long);
+  formData.append('phone', data?.phone);
+  formData.append('service_code', data.serviceCode);
+  formData.append('title', data?.title);
+  formData.append('media_file_1', {
+    uri: JSON.parse(data.images)[0]?.uri,
+    name: JSON.parse(data.images)[0]?.imageName,
+    type: JSON.parse(data.images)[0]?.mimeType
+  });
+  formData.append('media_file_2', {
+    uri: JSON.parse(data.images)[1]?.uri,
+    name: JSON.parse(data.images)[1]?.imageName,
+    type: JSON.parse(data.images)[1]?.mimeType
+  });
+  formData.append('media_file_3', {
+    uri: JSON.parse(data.images)[2]?.uri,
+    name: JSON.parse(data.images)[2]?.imageName,
+    type: JSON.parse(data.images)[2]?.mimeType
+  });
+  formData.append('media_file_4', {
+    uri: JSON.parse(data.images)[3]?.uri,
+    name: JSON.parse(data.images)[3]?.imageName,
+    type: JSON.parse(data.images)[3]?.mimeType
+  });
+  formData.append('media_file_5', {
+    uri: JSON.parse(data.images)[4]?.uri,
+    name: JSON.parse(data.images)[4]?.imageName,
+    type: JSON.parse(data.images)[4]?.mimeType
+  });
```

Comment:
what if there are more or less then 5? this should be looped over the amount of images, does the following work?

```
for (let i = 0; i < images.length; i++) {
  const image = images[i];

  formData.append(`media_file_${i + 1}`, {
    uri: image.uri,
    name: image.imageName,
    type: image.mimeType
  });
}
```

Reply:
Yes, that's right. I think this code will work. But I did the coding in this way because the API only allows 5 images. I will update it with a for loop, but I will still keep the image upload limit at 5.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c